website/CNAME

```diff
@@ -0,0 +1 @@
+pysangamam.org
```

Comment:
It looks like all contents have been moved to `website` folder. Please check this.